File: Modul 7/SOAL 5/Bahasa C/PRAK605-2210817210028-Muhammad Aulia Rasyid.c
#include <stdio.h>

int main (void)
{
    int ordo, a, b, c, isi, hasil;
    scanf ("%d", &ordo) ;
    int mat1[ordo][ordo], mat2[ordo][ordo] ;
    printf ("\nMatriks A\n") ;
    for (a = 0 ; a < ordo ; a++) {
        for (b = 0 ; b < ordo ; b++) {
            scanf ("%d", &isi) ;
            mat1[a][b] = isi ; } }
    printf ("\nMatriks B\n") ;
    for (a = 0 ; a < ordo ; a++) {
        for (b = 0 ; b < ordo ; b++) {
            scanf ("%d", &isi) ;
            mat2[a][b] = isi ; } }
    printf ("\nMatriks A x B\n") ;
    for (a = 0 ; a < ordo ; a++) {
        for (b = 0 ; b < ordo ; b++) {
            for (c = 0, hasil = 0 ; c < ordo ; c++) {
                hasil += mat1[a][c] * mat2[c][b] ; }
            printf ("%d ", hasil) ; }
        printf ("\n") ; }
}

File: Modul 7/SOAL 5/Bahasa Python/PRAK605-2210817210028-Muhammad Aulia Rasyid.py
ordo = int(input())
print("Matriks A")
mat1 = [] ; mat2 = []
for a in range(0, ordo) :
    isi1 = list(map(int, input().split()))
    mat1.append(isi1[0 : ordo])
print("\nMatriks B")
for a in range(0, ordo) :
    isi2 = list(map(int, input().split()))
    mat2.append(isi2[0 : ordo])
print("\nMatriks A x B")
for a in range(0, ordo) :
    for b in range(0, ordo) :
        hasil = 0
        for c in range(0, ordo) :
            hasil += mat1[a][c] * mat2[c][b]
        print(hasil, end=' ')
    print()